interfaces/cython/cantera/onedim.pyx

```diff
@@ -837,37 +837,51 @@ cdef class Sim1D:
         nPoints = [len(flow_domains[0].grid), 12, 24, 48]
 
         for N in nPoints:
+            if N > self.get_max_grid_points():
```

Comment:
I think it's because in practice, any instance of `Sim1D` is really a `FlameBase`, where this method has been overridden by the one that takes no arguments. I guess that's a downside to the pattern of having `FlameBase` override methods from `Sim1D` by filling in their obvious argument.